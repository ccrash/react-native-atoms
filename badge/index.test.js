import React from 'react'
import { configure, shallow } from 'enzyme'
import Adapter from 'enzyme-adapter-react-16'

import Badge from './index'

configure({ adapter: new Adapter() })

describe('Testing Badge component', () => {
  it('renders as expected', () => {
    const wrapper = shallow(<Badge number={0} />)
    expect(wrapper).toMatchSnapshot()
    wrapper.setProps({ number: 123 })
    expect(wrapper).toMatchSnapshot()
    wrapper.setProps({ number: -1 })
    expect(wrapper).toMatchSnapshot()
  })
})
