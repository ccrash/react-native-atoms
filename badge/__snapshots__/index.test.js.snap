// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing Badge component renders as expected 1`] = `ShallowWrapper {}`;

exports[`Testing Badge component renders as expected 2`] = `ShallowWrapper {}`;

exports[`Testing Badge component renders as expected 3`] = `ShallowWrapper {}`;
